pom.xml

```diff
@@ -292,7 +295,11 @@
             </properties>
         </profile>
         <profile>
+            <!-- this overrides java11 in java-parent-pom -->
             <id>java11</id>
+            <activation>
+                <jdk>[11,</jdk>
+            </activation>
```

Comment:
again would this conflict with the parent-pom one? should we give it a different name like we did in network?